tools/vpc-flowlogs-analysis/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+
+vpc_project_ids                = ["bigorg-hub-prod-prj-e126d530"]
```

Comment:
can you uncommit this file or make it terraform.tfvars.example and replace these job ids with place holders?